pkg/apps/konnector.go

```diff
@@ -22,6 +22,8 @@ type KonnManifest struct {
 	Editor     string `json:"editor"`
 	Icon       string `json:"icon"`
 
+	Agregator   *json.RawMessage `json:"agregator,omitempty"`
```

Comment:
In english, it is spelled with two `g`: aggregator ;-)

Can you add `cloned.Agregator = cloneRawMessage(m.Agregator)` in the `Clone` method below?